tests/TestCase/Controller/Component/RequestHandlerComponentTest.php

```diff
@@ -336,7 +337,7 @@ public function testViewClassMap()
     public function testIsAjaxParams()
     {
         $this->request->env('HTTP_X_REQUESTED_WITH', 'XMLHttpRequest');
-        $event = new Event('Controller.startup', $this->Controller);
+        $event = new ControllerEvent('Controller.startup', $this->Controller);
```

Comment:
I think's that can be a good idea but what if I don't want the prefix?

For example in case of tables.
When a user gets registered I'd like to send out a ```Users.registered``` event instead of ```Model.Users.registered```. It could be that I'm wrong and I should be using ```Model.Users.registered``` instead but that discussion is better suited for IRC I think.